css/style.css

```diff
@@ -9,285 +14,633 @@ body {
 }
 
 .bag__title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 16px;
-    line-height: 16px;
-    letter-spacing: .07em;
-    opacity: .5;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: .07em;
+  opacity: .5;
 }
 
 .bag-text{
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    opacity: .5;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  opacity: .5;
 }
 
 .product {
-    font-style: normal;
-    font-weight: 500;
-    font-size: 13px;
-    line-height: 16px; 
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  line-height: 16px; 
 }
 
 .bag-title-bold {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 40px;
-    line-height: 41px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
 }
 
 .bag-title {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 16px;
-    line-height: 16px;
-    letter-spacing: .07em;
-    opacity: .5;  
+  font-style: normal;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 16px;
+  letter-spacing: .07em;
+  opacity: .5;  
 }
 
 .total-look__text {
-    font-style: normal;
-    font-weight: normal;
-    font-size: 15px;
-    line-height: 21px;
-    opacity: .5;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 21px;
+  opacity: .5;
 }
 
 .name {
-    font-style: normal;
-    font-weight: bold;
-    font-size: 12px;
-    line-height: 15px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 15px;
 }
 
-.new-collection__background {
-    background-size: 100% auto;
-}
-
-
-    @media (max-width: 768px) { 
-        .logo,
-        .lang,
-        .check,
-        .new-сollection__social,
-        .text,
-        .new-сollection__main-card,
-        .new-collection__background {
-            display: none;
-            } 
-        } 
-
-        @media (min-width: 768px) { 
-        .logo-mobil,
-        .galka {
-            display: none;
-            } 
-        } 
-
-        @media (max-width: 768px) { 
-            .girl {
-                background-image: none;
-            }
-        }
+/* Cтили 2й секции ниже */
+
+.bag {
+  background: #FAE8E4;
+  height: 900px;
+  margin-top: 0;
+  width: 1440px;
+}
+
+.bag__wight {
+  position: absolute;
+  width: 398px;
+  height: 415px;
+  margin-left: 137px;
+  top: 1117px;
+}
+
+.bag__pink {
+  position: absolute;
+  width: 582px;
+  height: 844px;
+  margin-left: 231px;
+  top: 928px; 
+}
+
+.card {
+  position: absolute;
+  width: 488px;
+  height: 556px;
+  margin-left: 879px;
+  top: 1068px;
+  background: #FFFFFF;
+}
+.bag__title-bold {
+  position: absolute;
+  width: 400px;
+  height: 41px;
+  left: 55px;
+  top: 40px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 41px;
+  color: #000000;
+ }
+
+ .bag__title {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 0 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Love-metal_LOCO/pullRequest?prid=5449357)